units/stats_host_idle.go

```diff
@@ -180,15 +180,20 @@ func (j *collectHostIdleDataJob) incrementCostForDuration(ctx context.Context) (
 }
 
 func (j *collectHostIdleDataJob) getHostStatsMessage(cost float64, idleTime time.Duration) message.Composer {
+	taskHost := j.host.User == evergreen.User && !j.host.SpawnOptions.SpawnedByTask
+
 	// post host idle time message
 	msg := message.Fields{
-		"stat":         "host-idle",
-		"distro":       j.host.Distro.Id,
-		"provider":     j.host.Distro.Provider,
-		"provisioning": j.host.Distro.BootstrapSettings.Method,
-		"host_id":      j.host.Id,
-		"status":       j.host.Status,
-		"idle_secs":    idleTime.Seconds(),
+		"stat":            "host-idle",
+		"distro":          j.host.Distro.Id,
+		"provider":        j.host.Distro.Provider,
+		"provisioning":    j.host.Distro.BootstrapSettings.Method,
+		"host_id":         j.host.Id,
+		"status":          j.host.Status,
+		"idle_secs":       idleTime.Seconds(),
+		"spawn_host":      j.host.User != evergreen.User,
```

Comment:
This will include hosts spawned by tasks, since they have the user set to the task that spawned them.